atom/browser/atom_browser_context.cc

```diff
@@ -102,6 +119,16 @@ AtomBrowserContext::AtomBrowserContext(const std::string& partition,
   cookie_change_notifier_ = std::make_unique<CookieChangeNotifier>(this);
 
   BrowserContextDependencyManager::GetInstance()->MarkBrowserContextLive(this);
+
+#if BUILDFLAG(ENABLE_ELECTRON_EXTENSIONS)
+  BrowserContextDependencyManager::GetInstance()->CreateBrowserContextServices(
+      this);
+
+  extension_system_ = static_cast<extensions::AtomExtensionSystem*>(
+      extensions::ExtensionSystem::Get(this));
+  extension_system_->InitForRegularProfile(true /* extensions_enabled */);
```

Comment:
I think we might need to implement `AtomExtensionSystem::InitForIncognitoProfile` to fix tests failing at
```
[958:0428/030848.424091:FATAL:storage_frontend.cc(140)] Check failed: !browser_context_->IsOffTheRecord().
```